examples/showcase/web/manifest.json

```diff
@@ -12,12 +12,12 @@
         {
             "src": "icons/Icon-192.png",
             "sizes": "192x192",
-            "type": "image/png"
+            "_type": "image/png"
```

Comment:
I think that this file shouldn't be impacted by the change. You should revert the changes here.

Reply:
Fixed